wire-runtime-swift/src/test/swift/ProtoReaderTests.swift

```diff
@@ -105,48 +105,54 @@ final class ProtoReaderTests: XCTestCase {
     // MARK: - Tests - Decoding Messages And More
 
     func testDecodeBool() throws {
-        let data = Data(hexEncoded: "08_01")!
+        let data = Foundation.Data(hexEncoded: "08_01")!
         try test(data: data) { reader in
             let value = try reader.decode(tag: 1) { try reader.decode(Bool.self) }
             XCTAssertEqual(value, true)
         }
     }
 
     func testDecodeBytes() throws {
-        let data = Data(hexEncoded: """
+        let data = Foundation.Data(hexEncoded: """
             0A           // (tag 1 | Length Delimited)
             06           // Data length
             001122334455 // Random data
         """)!
         try test(data: data) { reader in
-            let value = try reader.decode(tag: 1) { try reader.decode(Data.self) }
-            XCTAssertEqual(value, Data(hexEncoded: "001122334455")!)
+            let value = try reader.decode(tag: 1) { try reader.decode(Foundation.Data.self) }
+            XCTAssertEqual(value, Foundation.Data(hexEncoded: "001122334455")!)
         }
     }
 
     func testDecodeMessage() throws {
-        let data = Data(hexEncoded: """
+        let data = Foundation.Data(hexEncoded: """
             0A       // (tag 1 | Length Delimited)
             04       // Length of name
             4C756B65 // Name value "Luke"
             10       // (tag 2 | Varint)
             05       // ID value 5
+            3A       // (tag 7 | Data)
+              04       // Length 4 for Data
```

Comment:
- [ ] Nit: weird alignment

Reply:
I do not think it is due to it being a new message type nested?